src/frontend/app/features/cloud-foundry/edit-space/edit-space-step/edit-space-step.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="edit-space">
+  <div>Specify the new name for the space (leave unchanged if don't wish to rename it): </div>
+  <form [formGroup]="editSpaceForm" class="edit-space__stepper-form">
```

Comment:
Needs `stepper-form`. The same kind if spacing should follow through to the ssh enabled flag